my_2d_robot/robo2d.launch

```diff
@@ -2,6 +2,7 @@
         <param name="robot_description" command="cat $(find my_2d_robot)/urdf/my2drobo.urdf" />
         <node name="robot_state_publisher" pkg="robot_state_publisher" type="robot_state_publisher" />
         <node name="my_2d_robo_state_publisher" pkg="my_2d_robot" type="my_2d_robot_node" />
+        <param name="set_loop_rate" type="double" value="2.0" />
```

Comment:
Rename param to just "loop_rate". We use verbs like "set" for function names since functions do something. Variable/param names should be nouns.